README.md

```diff
@@ -82,15 +84,14 @@ The list of currently supported accelerators (termed [Execution Providers](./doc
 
 ## API Documentation
 
-|Language|Supported Versions|Samples|
+|Language|Supported Version|Samples|
 |--|--|--|
-[Python](https://aka.ms/onnxruntime-python)| 3.5, 3.6, 3.7<br>[Python Dev Notes](./docs/Python_Dev_Notes.md)| [Samples](./samples#python)|
+[Python](https://aka.ms/onnxruntime-python)| 3.5, 3.6, 3.7<br>[Python Dev Notes](./docs/Python_Dev_Notes.md)|[Samples](./samples#python)|
```

Comment:
loss of space?

Reply:
looks like this is using an older version of the readme page. sorry, let me resync it to master and fix it.